From: Jeongjun Park <aha310510@gmail.com>
Subject: mm: percpu: fix typo to pcpu_alloc_noprof() description
Date: Wed, 25 Sep 2024 00:24:12 +0900

In the previous commit, the function name was changed from pcpu_alloc to
pcpu_alloc_noprof , but the function description was not changed
accordingly.  The function name should be changed to pcpu_alloc_noprof in
the description as well.

Link: https://lkml.kernel.org/r/20240924152412.118487-1-aha310510@gmail.com
Signed-off-by: Jeongjun Park <aha310510@gmail.com>
Cc: Christoph Lameter (Ampere) <cl@linux.com>
Cc: Dennis Zhou <dennis@kernel.org>
Cc: Tejun Heo <tj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/percpu.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/percpu.c~mm-percpu-fix-typo-to-pcpu_alloc_noprof-description
+++ a/mm/percpu.c
@@ -1726,7 +1726,7 @@ static void pcpu_alloc_tag_free_hook(str
 #endif
 
 /**
- * pcpu_alloc - the percpu allocator
+ * pcpu_alloc_noprof - the percpu allocator
  * @size: size of area to allocate in bytes
  * @align: alignment of area (max PAGE_SIZE)
  * @reserved: allocate from the reserved chunk if available
_
